test/specs/original/specs-spec.js

```diff
@@ -10,3 +11,13 @@ it('should run spec tests', function () {
     fail();
   }
 });
+
+it('should use the correct paragraph type', function () {
+  const md = 'A Paragraph.\n\n> A blockquote\n\n- list item\n';
+
+  const tokens = marked.lexer(md);
+
+  expect(tokens[0].type).toBe('paragraph');
```

Comment:
Think this makes more sense to be in the unit tests, yeah? The construction seems similar enough and this isn't, technically speaking, checking a specific spec or the output as a whole.